geomstats/information_geometry/binomial.py

```diff
@@ -10,9 +10,10 @@
 from geomstats.geometry.base import OpenSet
 from geomstats.geometry.euclidean import Euclidean
 from geomstats.geometry.riemannian_metric import RiemannianMetric
+from geomstats.information_geometry.information_manifolds import InformationManifold
```

Comment:
NIT: that would be information_manifold

We only use plural when the name used for the manifold is the name of its elements, e.g. THE manifold of binomial distributionS, which is however A Information Manifold.